shared/createAnApp/_default.md

```diff
@@ -1,4 +1,4 @@
-An application is a group of devices of the same type that all run the same code. When you provision a device, it is added to a specific application, but can be migrated to another application at any time.
+An application is a group of devices that run the same code and share the same architecture. When you provision a device, it is added to a specific application, but can be migrated to another application at any time.
```

Comment:
How about `that share the same architecture and run the same code`? This way the arch part comes as a complementary description of the initial statement instead of part of the features.